ompi/errhandler/errhandler.c

```diff
@@ -222,6 +231,165 @@ ompi_errhandler_t *ompi_errhandler_create(ompi_errhandler_type_t object_type,
   return new_errhandler;
 }
 
+#if OPAL_ENABLE_FT_MPI
+#include "opal/mca/threads/wait_sync.h"
+
+#if 0
```

Comment:
#if 0 to remove?

Reply:
We will want to activate this when we have the PRRTE changes integrated on the other side. We can bring this as a separate PR later though.